libs/sam-formly-sample/src/lib/feature/filter-wrapper/filter-wrapper.component.ts

```diff
@@ -3,13 +3,16 @@ import { FormGroup } from '@angular/forms';
 import { FormlyFormOptions, FormlyFieldConfig } from '@ngx-formly/core';
 import { BehaviorSubject } from 'rxjs';
 import { maxDateValidator, minDateValidator } from '@gsa-sam/sam-formly';
+
 //import { maxDateRangeValidator, minDateRangeValidator } from 'libs/packages/sam-formly/src/lib/formly/formly.validators';
 import { FormControl, ValidationErrors } from '@angular/forms';
+import { ValueConverter } from '@angular/compiler/src/render3/view/template';
 
 
 @Component({
-  selector: 'filter-wrapper',
+  selector: 'gsa-sam-filter-wrapper',

```

Comment:
Use sds prefix, this value must be wrong in tslint.

Reply:
fixed!